Configuration_adv.hpp

```diff
@@ -474,25 +480,40 @@
 
 // RA EndSwitch support
 //////////////////////////////////////////
+// End switches allow you to define how many percent of the distance needed to stop
```

Comment:
I would move this down to where `RA_ENDSWITCH_BACKSLEW_PERCENT` is defined (i.e. not at the top `USE_RA_END_SWITCH` block)